source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/index.html

```diff
@@ -84,14 +84,26 @@
       </p>
 
       <p>
-        From there, each email marketing solution charges a premium for sending through the user interface. With Legacy Newsletter, it's a <strong>$0.25 fee per 1,000 emails you send</strong>. For Marketing Campaigns, it's a <strong>$10 fee per 10,000 contacts you store</strong>. Your first 2,000 contacts in Marketing Campaigns are free.
+        From there, each email marketing solution charges a premium for sending through the user interface. With Legacy Newsletter, it's a <strong>$0.25 fee per 1,000 emails you send</strong>.
       </p>
   </div>
 </div>
 
-{% warning %}
-We will be offering Legacy Newsletter customers who migrate to Marketing Campaigns the option to pay $0.25/1,000 emails sent. This is currently in development with expected availability of June 1, 2017.
-{% endwarning %}
+You can continue to pay per email sent or you can choose to pay per contact you store in Marketing Campaigns:
+<table>
+  <tr>
+    <td>Pay **per email** you send</br>
+*Special offer for Legacy Newsletter migrators*</br>
+ </br>
+$0.25/1,000 emails</br>
+(plus base plan)</td>
+        <td>Pay **per contact** you store</br>
+*Standard Marketing Campaigns pricing*</br>
+ </br>
+$10/10,000 contacts</br>
+(plus base plan)</td>
+  </tr>
+</table>
 	
```

Comment:
Need more space between the bottom of the table and the buttons at the bottom of the page.